plugins/inputs/mysql/mysql.go

```diff
@@ -643,35 +826,56 @@ func (m *Mysql) gatherGlobalStatuses(db *sql.DB, serv string, acc telegraf.Accum
 			}
 
 			fields["slow_queries"] = i
+		case "Connections":
+			i, err := strconv.ParseInt(string(val.([]byte)), 10, 64)
+			if err != nil {
+				return err
+			}
+			fields["connections"] = i
+		case "Syncs":
+			i, err := strconv.ParseInt(string(val.([]byte)), 10, 64)
+			if err != nil {
+				return err
+			}
+			fields["syncs"] = i
+		}
+		// Send any remaining fields
```

Comment:
Hey @Maksadbek - this will need to move outside the above `for` loop or it ends up sending metrics multiple times (to match the implementation in `gatherGlobalVariables()`) :smile: